app/controllers/miq_ae_class_controller.rb

```diff
@@ -197,6 +200,20 @@ def get_node_info(node)
     x_history_add_item(:id => x_node, :text => @right_cell_text)
   end
 
+  def add_all_domains_version_message(domains)
+    @version_messages = []
+    domains.collect { |dom| [dom.name, dom.version, dom.available_version] }.each do |name, current, available|
+      @version_messages.push(_("%s domain: Current version - %s  Available version - %s") % [name, current, available]) if !current.nil? && !available.nil? && current != available
```

Comment:
Can we add a `,` between "Current" and "Available" like I had below:
`ManageIQ domain: Current version - 5.3, Available version - 5.4`